docs/introduction/Car-registry-tutorial.rst

```diff
@@ -169,25 +174,25 @@ This transaction allows us to buy a car or recall a car sell order.
             
         *Output boxes:*
         
-            Two possible outputs are possible. In the case of buying a car, new CarBox with a new owner, a new Regular box with a value declared in carBuyOrderInfo for the Car's former owner. 
+            Two possible outputs are possible. In the case of buying a car, new CarBox with a new owner, a new Regular box with a value declared in carBuyOrderInfo for the car's former owner. 
```

Comment:
@zennermac 

  Two outputs are possible. In the case of buying a car, a new CarBox with a new owner, a new RegularBox with a value declared in CarBuyOrderInfo for the car's former owner.